code/build.R

```diff
@@ -400,9 +400,9 @@ responce = system(command_tippecanoe, intern = TRUE)
 setwd("outputdata")
 # Check the combined_network_tile.geojson file is there:
 file.exists("simplified_network.geojson")
-command_tippecanoe = glue::glue("tippecanoe -o snet_{date_folder}.pmtiles",
-  "--name=snet",
-  "--layer=snet",
+command_tippecanoe = glue::glue("tippecanoe -o rnet_simplified_{date_folder}.pmtiles",
+  "--name=rnet_simplified",
```

Comment:
@mvl22 is this OK as a naming convention?

Slightly different from last time (_ not -) but should be quick fix when ready.